src/Microsoft.Azure.SignalR.Management/ClientEndpoint.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.SignalR.Management
+{
+    //todo public later
+    internal class ClientEndpoint
```

Comment:
this looks quite similar to the `NegotiationResponse`
https://github.com/Azure/azure-signalr/blob/a7fcd5c43556a88304d1821ebceb478738f1dd1e/src/Microsoft.Azure.SignalR/HubHost/NegotiateHandler.cs#L81-L87